### ISSUE TITLE
[solidity/en] Add a reminder that `constant` will be deprecated

### ISSUE THREAD

**Issue by varshard** [adambard/learnxinyminutes-docs#3065]
Please indicate that `constant` modifier will be deprecated and should be changed to `view` in the balance function of SimpleBank contract.

```solidity
  /// @notice Get balance
  /// @return The balance of the user
  // 'constant' prevents function from editing state variables;
  // allows function to run locally/off blockchain
  function balance() constant public returns (uint) {
        return balances[msg.sender];
    }
```

**Comment by prertik** [comment]
Do you want to send a pull request of this?

**Comment by divayprakash** [comment]
Closed in #3269
